shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/main/java/org/apache/shardingsphere/db/protocol/postgresql/constant/PostgreSQLMessageSeverityLevel.java

```diff
@@ -22,29 +22,29 @@
  *
  * @see <a href="https://www.postgresql.org/docs/12/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS">Table 19.2. Message Severity Levels</a>
  */
-public enum PostgreSQLMessageSeverityLevel {
+public class PostgreSQLMessageSeverityLevel {
```

Comment:
Please add `final` modifier and `@NoArgsConstructor(access = AccessLevel.PRIVATE)` here, and then we can merge this PR.